src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -495,8 +444,6 @@ private JsonObject buildInstanceRecordJsonObject(CompositePoLine compPOL, Map<St
                .stream()
                .map(pId -> {
                  JsonObject identifier = new JsonObject();
-                 identifier.put(INSTANCE_IDENTIFIER_TYPE_ID, productTypes.get(pId.getProductIdType()
```

Comment:
an instance cannot be created without an identifierTypeId, as this is a mandatory field https://github.com/folio-org/mod-inventory-storage/blob/master/ramls/instance.json#L79

Reply:
sure, I have reverted changes.